swap_meet/clothing.py

```diff
@@ -0,0 +1,18 @@
+from .item import Item
+
+class Clothing(Item):
+    """Class `Item`'s Child class.
+
+    Starts with default values for `condition` and `age`.
+    Initializes with a named attribute for its type, `"Clothing"`.
+    """
+
+    def __init__(self, condition=0, age=0):
+        super()
```

Comment:
This isn't actually doing anything as its written here.

Reply:
I think I get super() now! finallyyyy